item-spec/common-metadata.md

```diff
@@ -1,4 +1,6 @@
 # STAC Common Metadata
+This document outlines all commonly used fields that are currently supported in STAC Item properties. These fields are 
```

Comment:
```suggestion
This document outlines all commonly used fields for STAC Item properties. These fields are 
```

'that are currently supported' sounds a vit exclusive as if that are the only fields supported.